packages/ui/src/components/EasySetup/BalancesTransfer.tsx

```diff
@@ -2,7 +2,7 @@ import { Box, InputAdornment } from '@mui/material'
 import { styled } from '@mui/material/styles'
 import { SubmittableExtrinsic } from '@polkadot/api/types'
 import { ISubmittableResult } from '@polkadot/types/types'
-import GenericAccountSelection, { AccountBaseInfo } from '../GenericAccountSelection'
+import GenericAccountSelection, { AccountBaseInfo } from '../selectors/GenericAccountSelection'
```

Comment:
If found a bug here when you have a moderately long list of accounts, it doesn't get displayed entirely and we need to scroll to actually show it all, where it should be above.

![image](https://github.com/ChainSafe/Multix/assets/33178835/8935c296-b014-436a-a28c-529bcc98f42a)

Reply:
If you will start to scroll the content will move above the input. The behavior has changed due to the "disablePortal" props for Autocomplete. I disable it as it renders the models outside the "root" div and it was harder to customize the styles. 

<img width="955" alt="Screenshot 2023-07-11 at 15 39 03" src="https://github.com/ChainSafe/Multix/assets/8141917/44436a33-4547-4034-8ba4-6f77e216a46d">

Reply:
My list is very long, add some more accounts and you'll be able to reproduce. The positioning of the drop-down above or below is only related to the space available until the end of the screen, not to whether I scrolled or not. If the screen is high enough (btw I'm on a normal laptop, and it is) it's displayed below, and is cut.